docker-compose-e2e-statics.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+services:
+  nginx-server:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/nginx-server/Dockerfile
+    ports:
+      - 8081:80
+    networks:
+      - network1
+
+  playwright:
+    build:
+      context: .
+      dockerfile: ./dockerfiles/playwright/Dockerfile
+    command: yarn run test:visual:ci
+    environment:
+      BASE_URL: "http://nginx-server"
+      ipc: host
+    volumes:
+      - ./test/snapshots:/var/www/test/snapshots
+      - ./test-results:/var/www/test-results
+    depends_on:
+      - nginx-server
```

Comment:
There is no problem with container soft race without `HEALTHCHECK`?

Reply:
I haven't experienced it but it is possible so I added healtcheck